### DISCUSSION TITLE
Easing animation no longer works when going from version 3.2.1 to 3.3.0

### DISCUSSION THREAD

**Discussion by htobin** [jquery/jquery#5589] [Q&A]
Hello,

Apologies if this is already a discussion, but I didn't find anything in the search. I'm currently doing a jQuery library update for a project and there seems to be an issue with animate. After updating to 3.3.0, when using animate to scroll. Diving a little deeper I was able to add a console message that would appear for each step of the animation, and when the animate function is called with 3.3.0 there is a lack of steps in between the start of the scroll down.

Here is the function being called below
```
scrollContainer.animate({ scrollTop: stDestination},
	{duration: 1100, 
          easing: "swing",
          step:function(){
                  console.log("step");
          },
	  finish: function(){
              // do something
	  },
	  fail: function(){
	      console.error("scrolling to error failed");
	  }}
);
```
When triggering the scroll and comparing between the two versions, it appears that "step" log shows 69 times when using 3.2.1 and only  a single time when using 3.3.0. 

I would appreciate an explanation of what's going on and a work around if possible. Either of which are greatly appreciated. As I think this would be something to think about for other projects. Thanks!

**Comment by mgol** [comment]
Please provide a runnable test case on a platform like JS Bin, that really helps with reviewing it!
